app/views/landing_page/_hero.erb

```diff
@@ -19,11 +19,18 @@
   }
 <% end %>
 
+<%
+# Add non-breaking space to prevent widows
+subtitle_splitted = s["subtitle"]["value"].split(" ")
```

Comment:
At least hyphens might be simply replaced with their non-breaking versions: http://stackoverflow.com/a/7691604 but I don't know if there are some other characters still.